core/src/main/java/nl/nn/adapterframework/align/XmlTypeToJsonSchemaConverter.java

```diff
@@ -589,15 +655,29 @@ else if(facet == XSSimpleTypeDefinition.FACET_PATTERN){
 				}
 			}
 		}
+	}
 
+	private JsonObjectBuilder addType(JsonObjectBuilder builder, String type) {
+		return addType(builder, type, null);
 	}
 
-	private String getJsonDataType(short builtInKind){
-		String type;
+	private JsonObjectBuilder addType(JsonObjectBuilder builder, String type, XSParticle particle) {
+		builder.add("type", type);
+		if(particle != null) {
+			if(particle.getMinOccurs() >= 0) {
```

Comment:
```suggestion
			if(particle.getMinOccurs() > 0) {
```

Reply:
minOccurs defaults to 1. I think that minItems=0 should be displayed in the generated schema